crime-application-tracking/docker-compose.yml

```diff
@@ -7,3 +7,17 @@ services:
     ports:
       - "8490:8490"
       - "8500:8500"
+    depends_on:
+      - postgres
+
+  postgres:
+    image: postgres:14.3
```

Comment:
The Postgres version is managed by AWS so will not necessarily be this.

Reply:
This is for the local testing,  we are using CP version in other environments